File: sdk/dart/lib/model/shipment_item.dart
part of swagger.api;


@Entity()
class ShipmentItem {
  
  @Property(name: 'OrderID')
  String orderID = null;
  

  @Property(name: 'LineItemID')
  String lineItemID = null;
  

  @Property(name: 'QuantityShipped')
  int quantityShipped = null;
  
  ShipmentItem();

  @override
  String toString()  {
    return 'ShipmentItem[orderID=$orderID, lineItemID=$lineItemID, quantityShipped=$quantityShipped, ]';
  }

}

File: sdk/javascript-closure-angular/API/Client/ShipmentItem.js
goog.provide('API.Client.ShipmentItem');

/**
 * @record
 */
API.Client.ShipmentItem = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.ShipmentItem.prototype.orderID;

/**
 * @type {!string}
 * @export
 */
API.Client.ShipmentItem.prototype.lineItemID;

/**
 * @type {!number}
 * @export
 */
API.Client.ShipmentItem.prototype.quantityShipped;

